testenrichers/ejb-jakarta/src/main/resources/META-INF/services/org.jboss.arquillian.core.spi.LoadableExtension

```diff
@@ -0,0 +1 @@
+org.jboss.arquillian.testenricher.ejb.client.EJBEnricherExtension
```

Comment:
I know it's unlikely to have two enrichers at once in the classpath, but I wonder if we should keep the same package structure? It certainly will ease the migration of tests, but I wonder if that wouldn't potentially cause issues too - e.g. ppl importing wrong enriched or both at the same time. WDYT?

Reply:
Hmm from the top of my head, I can think of no situation where having both on classpath would not be user error. You either have test for project running `javax` or `jakarta`.

> ppl importing wrong enriched 

That's good point.

Reply:
I think it is that one would be using both versions of a given testenricher in a given project, but I can change the package if desired. No obvious best package jumps out at me. Options:
org.jboss.arquillian.testenricher.ejbjk.container
org.jboss.arquillian.testenricher.jakarta.ejb.container

Reply:
On second thought, I'd probably keep the packages identical. It is easier for migration and doesn't change anything about people importing wrong artifacts...

Reply:
Fair enough, also migration would be more about extensions relying on those rather than the actual "end-user" tests (but even with that SPI approach hides most of that). I think we can leave it as is.